app/controllers/v2/url_reservations_controller.rb

```diff
@@ -0,0 +1,15 @@
+class UrlReservationsController < ApplicationController
```

Comment:
Don't think we need this controller. It's not linked from the routes, in any case.